lib/client/index.js

```diff
@@ -28,6 +28,7 @@ client.init = function init(serverSettings, plugins) {
     , isInitialData = false
     , SGVdata = []
     , MBGdata = []
+    , pumpStatuses = []
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['pumpStatuses' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1181182976/issues/source?bid=2452379&fileBranchId=2921449#l31)